ti-lvgl-demo/suite/trixie/debian/control

```diff
@@ -0,0 +1,15 @@
+Source: ti-lvgl-demo
+Section: embedded
+Priority: optional
+Maintainer: Suhaas Joshi <s-joshi@ti.com>
+Build-Depends: debhelper-compat (= 13), cmake, libdrm-dev, libsdl2-dev, libsdl2-image-dev, mosquitto, mosquitto-dev, libmosquitto1, libmosquittopp1, libmosquitto-dev, mosquitto-clients, build-essential, libasound2t64, libasound2-dev, alsa-tools, alsa-utils, wayland-protocols, build-essential, ca-certificates
+Standards-Version: 4.5.0
+Homepage: https://github.com/texasinstruments/ti-lvgl-demo
+
+Package: ti-lvgl-demo
+Architecture: any
+Depends: ${misc:Depends}, ${shlibs:Depends}, libdrm-dev, libsdl2-dev, libsdl2-image-dev, mosquitto, mosquitto-dev, libmosquitto1, libmosquittopp1, libmosquitto-dev, mosquitto-clients, build-essential, libasound2t64, libasound2-dev, alsa-tools, alsa-utils, wayland-protocols, build-essential, ca-certificates
```

Comment:
Runtime depends should be automatically covered by just `${misc:Depends}, ${shlibs:Depends}`. No need to relist all these, nor add -dev pacakges, you don't need these at runtime.